src/hooks/cgroups/pbs_cgroups.PY

```diff
@@ -2738,14 +2739,16 @@ class CgroupUtils(object):
                         if int(entries[0]) not in pids:
                             pids.append(int(entries[0]))
             except:
-                raise
+                # PIDs may come and go as we read /proc so the glob data can
```

Comment:
Shouldn't this except be an except OSError?  The comment makes it sound like we're trying to be accepting of pids that go away.  That means the open will fail and raise an OSError.

Reply:
Added OSError.